app/activators.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+package app
+
+import (
+	"github.com/evmos/evmos/v20/app/eips"
+	"github.com/evmos/evmos/v20/x/evm/core/vm"
+)
+
+// EvmosActivators defines a map of opcode modifiers associated
+// with a key defining the corresponding EIP.
+var evmosActivators = map[string]func(*vm.JumpTable){
```

Comment:
nit: match the lowercase variable name also in the doc comments


```suggestion
// evmosActivators defines a map of opcode modifiers associated
// with a key defining the corresponding EIP.
var evmosActivators = map[string]func(*vm.JumpTable){
```